python/ray/autoscaler/aws/example.yaml

```diff
@@ -0,0 +1,65 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: default
+
+# The minimum number of workers nodes to launch in addition to the head
+# node. This number should be >= 0.
+min_workers: 0
+
+# The maximum number of workers nodes to launch in addition to the head
+# node. This takes precedence over min_workers.
+max_workers: 2
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-east-1
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ubuntu
+# By default Ray creates a new private keypair, but you can also use your own.
+# If you do so, make sure to also set "KeyName" in the head and worker node
+# configurations below.
+#    ssh_private_key: /path/to/your/key.pem
+
+# Provider-specific config for the head node, e.g. instance type. By default
+# Ray will auto-configure unspecified fields such as SubnetId and KeyName.
+# For more documentation on available fields, see:
+# http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
+head_node:
+    InstanceType: m5.large
+    ImageId: ami-212d465b
```

Comment:
which AMI is this?

Reply:
we should probably use the default Ubuntu AMI or the Amazon deep learning AMI (in the appropriate region)

Reply:
This is the deep learning AMI with Ray installed. We should make AMIs for each release probably (it's kind of a pain though since you have to replicate it to each region).

Reply:
Instead of making a new AMI, we could just run a `pip install` in the `head_init_commands` and `worker_init_commands` below. However that is less convenient for development

Reply:
Yeah that could work. Is it possible to pip install a particular travis build on an unmerged PR?